templates/plugin-template-js/index.js

```diff
@@ -1,18 +1,33 @@
+import { version } from "./package.json";
```

Comment:
the JS plugin won't have an import process I think? Need to manually enter version in this template I think?

Reply:
Does the import process require a build? I can change it but I was under the impression if there was package.json it would be updated with the latest versions as part of best practice.

Reply:
Manually entered and confirmed from testing that needs to be manually entered.